lib/jellyfish/room_service.ex

```diff
@@ -9,8 +9,8 @@ defmodule Jellyfish.RoomService do
 
   alias Jellyfish.{Event, Room, WebhookNotifier}
 
-  # in seconds
-  @metric_interval 10
+  # in milliseconds
+  @metric_interval 10 * Application.compile_env!(:jellyfish, :metrics_scrape_interval)
```

Comment:
The scrape interval is multiple of 10 milliseconds, so the default interval of `1000` corresponds to scraping each 10 seconds. I think it's a bit confusing. Maybe the `metrics_scrape_interval` env variable should represent milliseconds? We could add check preventing setting values lower than 10 ms.

Reply:
This interval is for WebRTC metrics, not Jellyfish metrics. It really shouldn't be used here. Please consult @mickel8 and @Karolk99 on whether we can and should make the interval for Jellyfish metrics configurable.